pkg/reconcilers/deploymentconfig.go

```diff
@@ -57,6 +57,77 @@ func GenericBackendMutators() []DCMutateFn {
 	}
 }
 
+func DeploymentConfigArgsMutator(_, existing *appsv1.DeploymentConfig) (bool, error) {
+	update := true
+	existing.Spec.Template.Spec.Containers[0].Args = []string{"bin/3scale_backend", "-s", "falcon", "start", "-e", "production", "-p", "3000", "-x", "/dev/stdout"}
+	return update, nil
+}
+
+func DeploymentConfigEnvMutator(desired, existing *appsv1.DeploymentConfig) (bool, error) {
+	update := false
+
+	// Always set env var CONFIG_REDIS_ASYNC to 1 this logic is only hit when you don't have logical redis db
+	for envId, envVar := range existing.Spec.Template.Spec.Containers[0].Env {
+		if envVar.Name == "LISTENER_WORKERS" {
+			if envVar.Value == "0" {
+				existing.Spec.Template.Spec.Containers[0].Env[envId].Value = "1"
+				update = true
+				return update, nil
```

Comment:
can we do a single update? 
In general I think it would be better to have specific mutator for worker and specific for listener instead of mixing them both.

Reply:
Not apposed to the idea but its a bit `DRY` in a coding sense.  Although I always find DRY code to be more readable .  Yea will pull it out.

Reply:
As for single update that means adding more update variables and an extra if  to check and only update when both are true, not sure the juice is worth the squeeze :thinking:

Reply:
rejigged ti for a single update